kong/conf_loader/constants.lua

```diff
@@ -562,9 +562,9 @@ local CONF_PARSERS = {
   error_template_xml = { typ = "string" },
   error_template_plain = { typ = "string" },
 
-  admin_gui_url = {typ = "string"},
-  admin_gui_path = {typ = "string"},
-  admin_gui_api_url = {typ = "string"},
+  admin_gui_url = { typ = "array" },
```

Comment:
We should also modify the `utils.prepare_variable` to support serializing the array to a string, or we will get something like `'ADMIN_GUI_URL': 'table: 0x011645f428',` in `kconfig.js`

Reply:
Good catch. I have fixed it locally.